### DISCUSSION TITLE
Feature request

### DISCUSSION THREAD

**Discussion by Galladean** [lnreader/lnreader#281] [Ideas]
I've been reading quite a bit of manhwa and have come to like the one tap scrolling method that a lot of websites and apps (like tachiyomi) use. The feature would be quite useful if adopted into lnreader. Also the current gesture control to swipe and switch between chapters is clunky and unusable because of the delay.

**Comment by rajarsheechatterjee** [comment]
There's already an open request for it #149. It is planned but there's no ETA for it.
